spatialos-sdk/examples/project-example/generated_code.rs

```diff
@@ -0,0 +1,1507 @@
+#[allow(unused_imports)]
+#[allow(unreachable_code)]
+#[allow(unreachable_patterns)]
+#[allow(unused_variables)]
+#[allow(dead_code)]
+#[allow(non_camel_case_types)]
+#[allow(unused_mut)]
+mod generated {
+    use spatialos_sdk::worker::component::*;
+    use spatialos_sdk::worker::internal::schema::*;
+    use std::collections::BTreeMap;
+
+    use super::generated;
+
+    /* Enums. */
+    /* Types. */
+    /* Components. */
+
+    pub mod example {
+        use spatialos_sdk::worker::component::*;
+        use spatialos_sdk::worker::internal::schema::*;
+        use std::collections::BTreeMap;
+
+        use super::super::generated;
+
+        /* Enums. */
+        /* Types. */
+        #[derive(Debug)]
+        pub struct CommandData {
+            value: i32,
+        }
+        impl TypeConversion for CommandData {
+            fn from_type(input: &SchemaObject) -> Result<Self, String> {
+                Ok(Self {
+                    value: input.field::<SchemaInt32>(1).get_or_default(),
+                })
+            }
+            fn to_type(input: &Self, output: &mut SchemaObject) -> Result<(), String> {
+                output.field::<SchemaInt32>(1).add(input.value);
+                Ok(())
+            }
+        }
+
+        /* Components. */
+
+        #[derive(Debug)]
+        pub struct Example {
+            x: f32,
```

Comment:
Looks like code generation is making all the fields of the generated structs private, should probably be public.

Reply:
oops. you're right.